modules/ingestor/src/graph/sbom/cyclonedx.rs

```diff
@@ -246,6 +266,7 @@ impl<'a> Creator<'a> {
                 .bom_ref
                 .as_ref()
                 .cloned()
+                // TODO: we might re-consider this
```

Comment:
elsewhere we generate uuid - here we seem to be ok using the name ?

Reply:
Yea, "todo". Didn't change the logic in this PR. Not happy about it. I'll take another look.

Reply:
Follow up PR: https://github.com/trustification/trustify/pull/1067